CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 ## 2.9.0 and 0.2.0 [BETA]
 - [java-security] and [spring-security] validates IAS OIDC tokens from multiple IAS tenants and zones.
 - [spring-security] Resource Server raises ``InvalidBearerTokenException`` in case token couldn't be successfully validated (as documented [here](https://docs.spring.io/spring-security/site/docs/current/reference/html5/#oauth2resourceserver-bearertoken-failure)).
+- The following dependencies are updated:
```

Comment:
The following dependencies **were** updated:

Reply:
see comment below...